core-services/testgrid-config-generator/_allow-list.yaml

```diff
@@ -36,20 +36,20 @@ periodic-ci-openshift-release-master-nightly-4.10-e2e-azure-csi-migration: broke
 periodic-ci-openshift-release-master-nightly-4.10-e2e-gcp-csi-migration: broken
 periodic-ci-openshift-release-master-nightly-4.11-e2e-aws-ovn-serial: informing
 release-openshift-origin-installer-e2e-azure-compact-4.4: broken
-periodic-ci-openshift-microshift-main-periodics-copr-rpm-health-check: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-api-machinery: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-apps: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-arch: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-auth: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-cli: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-instrumentation: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-network: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-node: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-scheduling: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-storage: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-reboot: informing
-periodic-ci-openshift-microshift-main-periodics-rebase-on-nightlies: informing
-periodic-ci-openshift-microshift-main-periodics-test-srpm: informing
-periodic-ci-openshift-microshift-main-periodics-test-unit: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-parallel: broken
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-serial: broken
+periodic-ci-openshift-microshift-main-periodics-copr-rpm-health-check: generic-informing
```

Comment:
why generic-informing instead of informing?

Reply:
The microshift test grid entries hadn't appeared yet and a look at the logs hinted at [why](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-prow-auto-testgrid-generator/1567664205102845952#1:build-log.txt%3A10-26).  The `unrecognized release type in job` message is generated by the testgrid-config-generator [L235-L261](https://github.com/openshift/ci-tools/blob/6fb323030b93edd8979c4863b697bba637760a0b/cmd/testgrid-config-generator/main.go#L235-L261).  If we want to rely on openshift-ci automation to generate the testgrid tabs for microshift, it'll need to be `generic-informining`.   Generic informing jobs can be found at https://testgrid.k8s.io/redhat-openshift-informing.

An alternative would be to alter the generator to create a separate tab for microshift.

Else, we can do what it looks like SNO and other projects have done: write/generate a config manually and open a PR directly into upstream testgrid, [here](https://github.com/kubernetes/test-infra/tree/master/config/testgrids/openshift).

I've not dealt with the testgrid before, so I'm open to whichever approach you'd recommend.

Reply:
We need to appear in https://testgrid.k8s.io/redhat-openshift-ocp-release-4.12-informing.  If that is gong to require a manual config, then that will have to be the way.